components/usage/pkg/db/conn.go

```diff
@@ -54,12 +55,25 @@ func ConnectForTests(t *testing.T) *gorm.DB {
 	})
 	require.NoError(t, err, "Failed to establish connection to DB. In a workspace, run `leeway build components/usage:init-testdb` once to bootstrap the DB.")
 
+	conn = conn.Debug()
+
 	t.Cleanup(func() {
+		// Delete records for known models from the DB
+		log.Info("Cleaning up DB between connections.")
+		for _, model := range []interface{}{
+			&WorkspaceInstance{},
+			&Workspace{},
+			&Project{},
```

Comment:
Do you want to pass in a struct of the type to be deleted to the `ConnectForTests` method, rather than iterating over known types here? This could be overlooked when adding new table.

Reply:
As in hoist these structs into a variable?